docs/parameters.yaml

```diff
@@ -1484,6 +1484,16 @@ default: 9.5s
 hidden: true
 components: ["cache"]
 ---
+name: Cache.TLSClientAuth
+description: |+
```

Comment:
It might be helpful to be more explicit about who "client" and "server" are here - caches are often the server, but they can also be a client.